src/components/LatestPosts.js

```diff
@@ -21,11 +21,11 @@ const POSTS = graphql`
   }
 `
 
-const Posts = ({ data, children }) => (
+const LatestPosts = ({ data, children }) => (
```

Comment:
I think calling it latestposts is more describing of what this component is doing